xml/Makefile.am

```diff
@@ -31,7 +31,7 @@ RNGdir			= $(dtddir)
 
 # Sorted list of available numeric RNG versions,
 # extracted from filenames like NAME-MAJOR[.MINOR][.MINOR-MINOR].rng
-RNG_numeric_versions    = $(shell ls -1 *.rng \
+RNG_numeric_versions    = $(shell echo $(wildcard $(top_srcdir)/xml/*.rng) \
```

Comment:
Looks like the problem is "ls -1" includes linefeeds where wildcard does not. I am in the process of rebasing and will submit a new PR